src/server/req_quejob.c

```diff
@@ -1707,36 +1710,29 @@ req_mvjobfile(struct batch_request *preq)
  *		Set state of job to JOB_STATE_QUEUED (or Held or Waiting) and
  *		enqueue the job into its destination queue.
  *
- *  @param[in]	preq	-	The batch request structure
+ * @param[in]	preq	-	The batch request structure
+ * @param[in]	pj		-   Pointer to the job structure
  *
  */
-
 void
-req_commit(struct batch_request *preq)
+req_commit_now(struct batch_request *preq,  job *pj)
```

Comment:
For jobs without scripts, does it make sense in req_commit_now to even try to save job script to db?

Reply:
Yes makes sense - i always thought i had a check before writing the job script - turns out i don't - adding now